README.md

```diff
@@ -61,6 +76,91 @@ For a full example of `build.sbt` you can have a look at this [build.sbt](https:
 used for the example.
 
 
+## Use PureConfig
+
+Import the library and use one of the `loadConfig` methods:
+
+
+```scala
+import pureconfig._
+
+val config: Try[YourConfClass] = loadConfig[YourConfClass]
+```
+
+
+## Supported Types
+
+Currently supported types for fields are:
+- `String`, `Boolean`, `Double` (standard
+and percentage format ending with `%`), `Float` (also supporting percentage),
+`Int`, `Long`, `Short`, `URL`, `Duration`, `FiniteDuration`
+- all collections implementing the `TraversableOnce` trait where the elements type
+is in this list
+- `Option` for optional values, i.e. value that can or cannot be in the configuration
+- `Map` with `String` keys and any value type that is in this list
+- typesafe `ConfigValue`, `ConfigObject` and `ConfigList`.
```

Comment:
I'm probably nitpicking, but the period at the end of this line seems a bit off, specially since there are two more bullet points after.